master/introduction/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: About Calico
+---
+
+Calico provides secure networking for containers and
+other distributed, dynamic workloads.
+
+Using the Linux kernel and BGP, Calico creates and manages 
+a flat virtual network, allowing any node to route to any other. 
```

Comment:
do you mean workload (e.g. pod or VM) rather than node?

Reply:
Thanks @caseydavenport ! I actually have some updated content I should be pushing up shortly. But yes, we are moving to "workload".